apps/mull-ui/src/graphql/queries.gql

```diff
@@ -48,3 +48,21 @@ query GetParticipatingEvents($participatingEventsUserId: Int!) {
     title
   }
 }
+
+# Find events by Id and finds participating events of user
+query EventPage($eventId: Int!, $participatingEventUserId: Int!) {
+  participatingEvent(eventId: $eventId, userId: $participatingEventUserId)
+
+  event(id: $eventId) {
+    ...EventContent
+  }
+}
+
+fragment EventContent on Event {
+  id
+  title
+  description
+  startDate
+  endDate
+  restriction
+}
```

Comment:
Why was this added? Just curious

Reply:
It's not needed, but I think it would be cleaner if any new queries require the same event content, it can use this fragment

Reply:
You can leave it for now, but I think that kind of refactoring where you take our duplicates into a var or something should be done on a need basis.